### DISCUSSION TITLE
How to use the HTML format

### DISCUSSION THREAD

**Discussion by laoshanxi** [grafana/grafana-infinity-datasource#481] [Q&A] [answered]
For [HTML format](https://sriramajeyam.com/grafana-infinity-datasource/wiki/html)

Does that mean the requested URL should return full HTML page (include header, body) like below, not segment, right?

https://github.com/yesoreyeram/grafana-infinity-datasource/blob/main/testdata/users.html

**Comment by yesoreyeram** [comment]
Yes correct..

**Reply by laoshanxi** [reply]
> es correct..

Thanks for the reply, but when I used like "https://www.google.com" as the URL, it cannot work, is there any different with this static html?

**Reply by laoshanxi** [reply]
> Yes correct..

If you can provide a configure example for this test data that would help a lot:
https://github.com/yesoreyeram/grafana-infinity-datasource/blob/main/testdata/users.html

**Comment by laoshanxi** [comment]
and also when I use the sample HTML, I faced this error: 

![image](https://user-images.githubusercontent.com/50568160/205056056-6adc68f6-5304-4671-8100-d9aa67595fd5.png)

so I really do not understand how this format will work.

**Comment by laoshanxi** [comment, marked as answer]
Understand now, HTML format can not show RAW content, but only table data from HTML attributes:

https://app-mesh.readthedocs.io/en/latest/GrafanaDataSource.html#grafana-infinity-datasource
